src/components/Select/index.js

```diff
@@ -145,6 +152,7 @@ Select.defaultProps = {
     onClick: () => {},
     onFocus: () => {},
     onBlur: () => {},
+    bottomHelpText: undefined,
```

Comment:
in order to be more consistent, let's use `null` instead `undefined` as default prop